spec/requests/api/chargebacks_spec.rb

```diff
@@ -69,7 +69,137 @@
     expect_request_success
   end
 
+  it "can list of all currencies" do
+    currency = FactoryGirl.create(:chargeback_rate_detail_currency_EUR)
+
+    api_basic_authorize
+    run_get '/api/currencies'
+
+    expect_result_resources_to_include_hrefs(
+      "resources", ["/api/currencies/#{currency.id}"]
+    )
+    expect_result_to_match_hash(response_hash, "count" => 1)
+    expect_request_success
+  end
+
+  it "can show an individual currency" do
+    currency = FactoryGirl.create(:chargeback_rate_detail_currency_EUR)
+
+    api_basic_authorize
+    run_get "/api/currencies/#{currency.id}"
+
+    expect_result_to_match_hash(
+      response_hash,
+      "name" => currency.name,
+      "id"   => currency.id,
+      "href" => "/api/currencies/#{currency.id}"
+    )
+    expect_request_success
+  end
+
+  it "can list of all measures" do
+    measure = FactoryGirl.create(:chargeback_rate_detail_measure_bytes)
+
+    api_basic_authorize
+    run_get '/api/measures'
+
+    expect_result_resources_to_include_hrefs(
+      "resources", ["/api/measures/#{measure.id}"]
+    )
+    expect_result_to_match_hash(response_hash, "count" => 1)
+    expect_request_success
+  end
+
+  it "can show an individual measure" do
+    measure = FactoryGirl.create(:chargeback_rate_detail_measure_bytes)
+
+    api_basic_authorize
+    run_get "/api/measures/#{measure.id}"
+
+    expect_result_to_match_hash(
+      response_hash,
+      "name" => measure.name,
+      "id"   => measure.id,
+      "href" => "/api/measures/#{measure.id}",
+    )
+    expect_request_success
+  end
+
   context "with an appropriate role" do
+    it "can create a new chargeback rate" do
+      api_basic_authorize action_identifier(:chargebacks, :create, :collection_actions)
+
+      expect do
+        run_post chargebacks_url,
+                 :description => "chargeback_0",
+                 :rate_type   => "Storage"
+      end.to change(ChargebackRate, :count).by(1)
+      expect_result_to_match_hash(response_hash["results"].first, "description" => "chargeback_0",
+                                                                  "rate_type"   => "Storage",
+                                                                  "default"     => false)
+      expect_request_success
+    end
+
+    it "returns bad request for incomplete chargeback rate" do
+      api_basic_authorize action_identifier(:chargebacks, :create, :collection_actions)
+
+      expect do
+        run_post chargebacks_url,
+                 :rate_type   => "Storage"
+      end.not_to change(ChargebackRate, :count)
+      # expect_bad_request(/description can't be blank/i)
+    end
+
+    it "can edit a chargeback rate through POST" do
+      chargeback_rate = FactoryGirl.build(:chargeback_rate, :description => "chargeback_0")
+      chargeback_rate.save
```

Comment:
I believe you can consolidate these two lines by using `create` instead of `build`